front/src/reducers/osrdconf/operationalStudiesConf/utils.ts

```diff
@@ -83,7 +83,7 @@ export const addPathStep = (pathSteps: PathStep[], newPathStep: PathStep): PathS
     const index = pathSteps.findIndex(
       (step) => step.positionOnPath && step.positionOnPath > newPathStep.positionOnPath!
     );
-    return addElementAtIndex(pathSteps, index, newPathStep);
+    return addElementAtIndex(pathSteps, index, { ...newPathStep, isFromPowerRestriction: true });
```

Comment:
This flag will be set when the path step is created, but when reloading the page this won't work anymore right?

Reply:
Yes, it won't work anymore, but it's the best way I found to avoid removing by mistake pathSteps which should not be removed :/

Reply:
This will only result in false negatives, ie. after page reload path steps related to power restrictions won't be cleared. Since there is no data loss, that's fine.